techniques/failures/F109.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html><html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
+ 
+ <head><title>Failure of Success Criterion 3.3.8 and 3.3.9 due to forcing transcription of individual password characters</title>
```

Comment:
Is this title just saying "Failure of .... due to requiring manual entry of passwords"? Would that be simpler?